src/hooks/useTransactionHistory.ts

```
@@ -0,0 +1,175 @@
+
+        // Convert ISO strings to Date objects for timestamps
+        return {
+            ...data,
+            entries: data.entries.map((entry: HistoryEntry) => {
+                const extraData = entry.extraData ?? {}
+                let link: string
+                let tokenSymbol: string
+                let usdAmount: string
+                switch (entry.type) {
+                    case 'SEND_LINK':
+                        const password = getFromLocalStorage(`sendLink::password::${entry.uuid}`)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)